frontend/src/common/Footer.js

```diff
@@ -0,0 +1,91 @@
+import React, {Component } from 'react';      
+
+class PageFooter extends Component {
+  constructor(props) {
+    super(props)
+    // this.state = {
+    //   none: ''
+    // }
+  }
```

Comment:
Circle CI doesn't like that you don't have anything in `constructor()`:
```
./src/common/Footer.js
  Line 4:  Useless constructor  no-useless-constructor
```

See here: https://circleci.com/gh/the-difference-engine/rescue-leftover-cuisine2/124